feature_tests/src/attrs.rs

```diff
@@ -88,25 +88,25 @@ pub mod ffi {
     pub struct MyIndexer(Vec<String>);
```

Comment:
should also be `auto`, as all other `any(_)`s

Reply:
I don't think `auto` works that way

Reply:
I thought that's the point?

Reply:
However I should use real `not(any(supports ...))`s here instead

Reply:
ah yeah this is disable

Reply:
@robertbastian No, it's only "auto" based on the actual attribute being used. It won't affect `disable`

Reply:
I plan to document this in the diplomat book later when I'm back, but the idea is that `auto` saves you from doing redundant `attr(supports = foo, foo)`. If you're doing something more complicated, like "this type only exists for backends with iterators", you still need to do `not(supports = iterators), disable` etc. Which was why I resurrected these supports attributes in https://github.com/rust-diplomat/diplomat/pull/584

Reply:
there are still explicit backends in this file, please recover them as well. Might want to do `git reset` to f0f1a1a746fa87020bb7b7397d92ba0c60db6fc7 and work from there.

Reply:
Looks like this test is currently disabled in all backends anyway. I'm going to leave it in for when JS gets indexer support, but it appears to be that Dart indexer support is broken around indexers that return strings.

Reply:
How so?

Reply:
https://github.com/rust-diplomat/diplomat/issues/590

Reply:
It's unrelated to indexers, it's about returning `&DiplomatStr`.